packages/eas-cli/src/rollout/actions/EndRollout.ts

```diff
@@ -200,21 +192,21 @@ export class EndRollout implements EASUpdateAction<UpdateChannelBasicInfoFragmen
     }
     const { rolledOutBranch, defaultBranch } = rollout;
     Log.newLine();
-    if (selectedOutcome === EndOutcome.REPUBLISH_AND_ROUTE_BACK) {
+    if (selectedOutcome === EndOutcome.REPUBLISH_AND_SEND_BACK) {
       Log.log(`Ending the rollout will do the following:`);
       const actions = formatFields([
         {
           label: '1.',
-          value: `🔁 Republish the Update Group from ${chalk.bold(
+          value: `🔁 Republish the update group from ${chalk.bold(
             rolledOutBranch.name
           )} onto ${chalk.bold(defaultBranch.name)}`,
         },
-        { label: '2.', value: `⬅️  Route all users back to ${chalk.bold(defaultBranch.name)}` },
+        { label: '2.', value: `⬅️  Send all users back to ${chalk.bold(defaultBranch.name)}` },
```

Comment:
I'm having trouble trying to figure out words. I definitely prefer "send" over "route", but here's some other ones I like:
- `Put all users back on main`
- `Revert all users back to main` (this one would need an opposite of revert for the other sentence, i.e. `What percent of users should <branch_name> be rolled out to?`)

I think I prefer revert/roll out.

Reply:
ok. I've gone with revert/roll out where it makes sense, and send otherwise. For example, it makes sense to say `revert users back to main` or `roll out to staging`. But here's a case where it makes sense to use 'send': `10% of users are rolled out to main and 90% of users are sent to staging`

Reply:
`10% of users remain on main and 90% of users are rolled out on staging` (or vice versa, not sure which direction the roll out was)